integrations/extensions/docs/elasticsearch-install-and-setup/how_to_use_web_crawler_in_elasticsearch.md

```diff
@@ -0,0 +1,332 @@
+  * `_ingest._value.sparse` expects a `sparse` field for each chunk object as the target field.
+  * `_ingest._value.text` expects a `text` field for each chunk object as the input field.
+  * `search-crawler-with-chunking@custom` is the name of the ingest pipeline. You need to update it with your ingest pipeline name.  
+  
+
+  > ⛔️
+  > **Caution**  
+  > Don't forget to click on `Save pipeline` to save your changes!
+
+
+* Start your web crawler and monitor its progress  
+  Once you have added the processors to your ingest pipeline, you can kick off your web crawler to crawl the website URLs you have configured at earlier steps.  
+
+  If you already started your web crawler at the end of [Step 2](#step-2-create-and-configure-a-web-crawler-in-elasticsearch), 
+  and it finished extracting the documents, you may need to delete all the documents first with the following cURL command:
+  ```shell
+  curl -k -X POST "${ES_URL}/${ES_INDEX_NAME}/_delete_by_query" \
+  -u "${ES_USER}:${ES_PASSWORD}" \
+  -H 'Content-Type: application/json' -d'
+  {
+     "query":{
+       "match_all":{}
+     }
+  }'
+  ```
+  Then, you can follow these steps to start your web crawler with the custom ingest pipeline:
+  * Go to your web crawler index page, and click on `Crawl` in the upper right corner to start it. 
+  * You will see new crawl requests on the overview page, and you can click on the request ids to see more details and to monitor the progress of your crawl requests.
+    <img src="assets/web_crawler_overview_with_crawl_requests.png" width="966" height="563" />
+  * If you see your crawler is running and the number of documents is increasing, your web crawler is working with the ingest pipeline. 
+    You can now inspect the documents and learn more about the web crawler from [Elastic documentation](https://www.elastic.co/guide/en/enterprise-search/8.10/crawler.html) to improve or customize your web crawler. 
+
+
+* Run a nested `text_expansion` query using cURL
+  ```shell
+  curl -k -X GET "${ES_URL}/${ES_INDEX_NAME}/_search?pretty" \
+  -u "${ES_USER}:${ES_PASSWORD}" \
+  -H 'Content-Type: application/json' -d'
+  {
+    "query": {
+      "nested": {
+        "path": "passages",
+        "query": {
+          "text_expansion": {
+            "passages.sparse.tokens": {
+              "model_id": ".elser_model_1",
+              "model_text": "what is the best time to visit Arcadia national park?"
+            }
+          }
+        },
+        "inner_hits": {"_source": {"excludes": ["passages.sparse"]}}
+      }
+    }
+  }'
+  ```
+  The above command sends a nested query to the Elasticsearch index. `text_expansion` is used in this query on the ELSER tokens 
+  generated for each chunked text by the ingest pipeline. So, the search happens on the chunked texts. Learn more about nested 
+  query from the [Elastic documentation](https://www.elastic.co/guide/en/elasticsearch/reference/8.10/query-dsl-nested-query.html).
```

Comment:
I would also add a note saying that if you run this query while the crawling is taking place, you might get a timeout error, because the ELSER model is busy indexing and thus might not respond quickly enough to your query.  If that happens, the user should just wait until the crawl finishes.